src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/cdk_stacks/tests/test_pipeline_creation.py

```diff
@@ -140,3 +140,54 @@ def test_pipeline_creation_outputs_as_expected_when_source_is_codecommit_and_bui
     assert build_stage_action['ActionTypeId']['Provider'] == "CodeBuild"
 
     assert len(build_stage['Actions']) == 1
+
+def test_pipeline_creation_outputs_as_expected_when_source_is_codecommit_with_codebuild_clone_ref_and_build_is_codebuild():
```

Comment:
Can you update the test above this one to test that the `source_stage_action['Configuration']['OutputArtifactFormat']` is not set?